.eslintrc.js

```diff
@@ -44,5 +44,11 @@ module.exports = {
         'max-lines-per-function': 'off',
       },
     },
+    {
+      files: ['packages/config/src/validate/validations.js'],
+      rules: {
+        'max-lines': 'off',
```

Comment:
Since this is for a single file, should we use an inline comment instead?

```js
/* eslint-disable max-lines */

...

/* eslint-enable max-lines */
```

This would prevent forgetting this when the filename is renamed.

Reply:
Sure! Done in https://github.com/netlify/build/pull/2390/commits/7a3ef300c36e1376c07f97b8d8df88ef8c83e7d2.